onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1066,167 +1065,186 @@ def executor(function, args, kwargs):
     return executor
 
 
-def _get_test_class_name(cls, num, params_dict) -> str:
-    del cls  # unused
-    del num  # unused
-    return params_dict["name"]
-
-
-@parameterized.parameterized_class(
-    [
-        {
-            "function_executor": _eager_executor,
-            "name": "TestOutputConsistency_Eager",
-        },
-        {
-            "function_executor": _graph_executor,
-            "skip_test": unittest.SkipTest("only torch>=2.0 is supported")
-            if version_utils.torch_older_than("2.0")
-            else None,
-            "name": "TestOutputConsistency_FullGraph",
-        },
-    ],
-    class_name_func=_get_test_class_name,
-)
-class TestOutputConsistency(unittest.TestCase):
-    """Test output consistency between exported ONNX models and PyTorch eager mode.
+def run_test_output_match(
+    test_suite: unittest.TestCase,
+    device: str,
+    dtype: torch.dtype,
+    op,
```

Comment:
nit type annotation

Reply:
if it's not an `OpInfo`, perhaps consider creating a Protocal

Reply:
Done